pkg/cmd/controller/standalone_apiserver.go

```diff
@@ -62,7 +62,7 @@ func RunControllerServer(servingInfo configv1.HTTPServingInfo, kubeExternal clie
 
 	//handler = apifilters.WithAuthentication(handler, authn, apifilters.Unauthorized(legacyscheme.Codecs, false), nil)
 	handler = apifilters.WithAuthentication(handler, authn, apifilters.Unauthorized(serializer.WithoutConversionCodecFactory{CodecFactory: legacyscheme.Codecs}), nil)
-	handler = apiserverfilters.WithPanicRecovery(handler)
+	handler = apiserverfilters.WithPanicRecovery(handler, requestInfoResolver)
```

Comment:
This checks out - see https://github.com/kubernetes/apiserver/blob/2cbad6fc8711ad50641e321a630a152caaf01623/pkg/server/config.go#L759